### ISSUE TITLE
Suggestion to create tags inline with cursor in between like in HTML

### ISSUE THREAD

**Issue by everspader** [vscode-django/vscode-django#54]
As a suggestion for future releases, it would be nice if the tags were created in the same line just as the HTML ones like
`{% block content %}{% endblock content %}`
With the cursor in between so if the user decides to use multiline then by simply hitting 'enter' you'd get the nice indent
```
{% block content %}    
    something
{% endblock content %}
```

**Comment by ZackPlauche** [comment]
Anyone know what it would take to move this forward?

**Comment by batisteo** [comment]
Quick question, isn’t the snippet `blok` enough?

**Comment by batisteo** [comment]
Anyway, fixed by a7a7725 in `1.1.0`
